addon/webextension/content-script.js

```diff
@@ -90,9 +98,15 @@ function wrapWith (element, config) {
     // optional: Accept node always.  Same a 'no function' here.
     { acceptNode: function(node) {
       // Logic to determine whether to accept, reject or skip node
-      return NodeFilter.FILTER_ACCEPT;
-      //
+
+      // skip scripts and styles
+      const tag = node.parentElement.tagName;
+      const skipped_tags = ["STYLE", "SCRIPT", "CANVAS", "SVG"]
+      if skipped_tags.includes(tag) {
```

Comment:
This is invalid; need to add a parentheses around the condition statement for multiline `if` statements.